metadata-ingestion/src/datahub/ingestion/source/dremio/dremio_config.py

```diff
@@ -0,0 +1,242 @@
+import datetime
+import os
+from typing import List, Literal, Optional
+
+import certifi
+import pydantic
+from pydantic import Field, validator
+
+from datahub.configuration.common import AllowDenyPattern, ConfigModel
+from datahub.configuration.source_common import (
+    EnvConfigMixin,
+    PlatformInstanceConfigMixin,
+)
+from datahub.ingestion.source.ge_profiling_config import GEProfilingConfig
+from datahub.ingestion.source.state.stale_entity_removal_handler import (
+    StatefulStaleMetadataRemovalConfig,
+)
+from datahub.ingestion.source.state.stateful_ingestion_base import (
+    StatefulIngestionConfigBase,
+)
+from datahub.ingestion.source.usage.usage_common import BaseUsageConfig
+from datahub.ingestion.source_config.operation_config import is_profiling_enabled
+
+
+class DremioConnectionConfig(ConfigModel):
+    hostname: Optional[str] = Field(
+        default=None,
+        description="Hostname or IP Address of the Dremio server",
+    )
+
+    port: int = Field(
+        default=9047,
+        description="Port of the Dremio REST API",
+    )
+
+    username: Optional[str] = Field(
+        default=None,
+        description="Dremio username",
+    )
+
+    authentication_method: Optional[str] = Field(
+        default="PAT",
+        description="Authentication method: 'password' or 'PAT' (Personal Access Token)",
+    )
+
+    password: Optional[str] = Field(
+        default=None,
+        description="Dremio password or Personal Access Token",
+    )
+
+    tls: bool = Field(
+        default=True,
+        description="Whether the Dremio REST API port is encrypted",
+    )
+
+    disable_certificate_verification: Optional[bool] = Field(
+        default=False,
+        description="Disable TLS certificate verification",
+    )
+
+    path_to_certificates: str = Field(
+        default=certifi.where(),
+        description="Path to SSL certificates",
+    )
+
+    is_dremio_cloud: bool = Field(
+        default=False,
+        description="Whether this is a Dremio Cloud instance",
+    )
+
+    dremio_cloud_region: Literal["US", "EMEA"] = Field(
+        default="US",
+        description="Dremio Cloud region ('US' or 'EMEA')",
```

Comment:
This should be
description="Dremio Cloud region ('US' or 'EU')",

https://docs.dremio.com/cloud/reference/api/#api-endpoint